server/client_manager.py

```diff
@@ -2107,3 +2148,30 @@ def get_multiclients(self, ipid=-1, hdid=""):
 
     def get_mods(self):
         return [c for c in self.clients if c.is_mod]
+        
+    class Battle_char:
```

Comment:
this breaks the naming scheme - name it "BattleChar"

Reply:
Done!